internal/database/mariadb/issue_match_test.go

```diff
@@ -341,6 +341,75 @@ var _ = Describe("IssueMatch", Label("database", "IssueMatch"), func() {
 					}
 
 				})
+				It("can filter issue matches by PrimaryName using wild card search", func() {
```

Comment:
Is this copied from here: https://github.com/cloudoperators/heureka/blob/60dfd0af5d4c0abc4e5451024921b10de157e45a/internal/database/mariadb/issue_test.go#L329
It seems to be duplicated.

Reply:
@dustindemmerle ☝️